### ISSUE TITLE
Incorrect syntax errors

### ISSUE THREAD

**Issue by JihadiJackass** [cwtools/cwtools-vscode#38]
Hey, so I'm making a formables mod, and I noticed every single one of my decisions has errors which I've checked, and a lot of it is stuff that works, but the editor doesn't expect it, which leaves me with lots of false-errors.

Can I please have an option to disable certain errors?

![image](https://user-images.githubusercontent.com/11571144/87360176-8694e700-c537-11ea-9018-fc9ebc8751f5.png)
This works and is how I prefer it, I would like to disable this notification.

![image](https://user-images.githubusercontent.com/11571144/87360204-9c0a1100-c537-11ea-8dff-3ab37d6c37fe.png)
Same here, Idk why it's telling me I'm not grabbing the ideas, because I am. Would like to disable this notification as well.

![image](https://user-images.githubusercontent.com/11571144/87360239-ae844a80-c537-11ea-8cf7-ecc8787e2ed8.png)
Basically all of these errors are stuff that functions as I want it to, but still tells me is a problem.

Please allow us to right click -> hide similar alerts, or something like this.

**Comment by noah1gamble** [comment]
![image](https://user-images.githubusercontent.com/11571144/87581871-a1866900-c6a7-11ea-9db8-46aaa4421a12.png)

Another instance where I would like to disable to notification. This is a cheat mod, I don't want players clicking this button 8 times, when they only have to click it once for 800.

**Comment by maxice8** [comment]
> Can I please have an option to disable certain errors?

There is a `cwtools.errors.ignore` configuration key that you can edit, here is what I use since my mod doesn't localise modifier description.

```
"cwtools.errors.ignore": [
	"CW100",
    "CW246"
]
```

> Another instance where I would like to disable the notification. This is a cheat mod, I don't want players clicking this button 8 times, when they only have to click it once for 800.

I would recommend against it, that notification happens because the configuration files that CWTools reads to understand how to read Europa Universalis IV code (there are equivalents for Hearts of Iron, Stellaris, etc) is wrong, it **should** accept any integer value that the game engine also accepts, I think when someone initially added that `<effect>` in the configuration files for CWTools to recognize they assumed it had that limit.
